README.rst

```diff
@@ -1,95 +1,167 @@
-=========
+#########
 I Promise
-=========
+#########
+
 .. image:: https://badge.fury.io/py/ipromise.svg
     :target: https://badge.fury.io/py/ipromise
 
-This repository provides a Python base class, and various decorators for specifying promises relating to inheritance.
-It provides three inheritance patterns:
+This repository, ``ipromise``, provides a Python base class and
+decorators to guarantee inheritance "promises".
+
+Three inheritance patterns are provided:
+
+* implementing
+* overriding
+* augmenting
+
+Using the ``ipromise`` inheritance patterns can ensure an inheritance hierarchy
+is used as intended by forcing a build-time or run-time failure when not.
+
+----
+
+.. contents::
+
+----
+
+**********
+Installing
+**********
 
-* implementing,
-* overriding, and
-* augmenting.
+.. code-block:: shell
 
-Base class
-==========
-Checking promises depends on inheritance from the base class ``AbstractBaseClass``.  Unlike the standard library's similar class ``abc.ABCMeta``, ``AbstractBaseClass`` does not bring in any metaclasses.  This is thanks to Python 3.6's PEP 487, which added ``__init_subclass__``.
+    python -m pip install ipromise
```

Comment:
I like this section, but if you're doing this, your pip isn't configured property.  Just write "pip install ipromise".

Reply:
Fixed.